cpp/src_prims/timeSeries/jones_transform.h

```diff
@@ -0,0 +1,483 @@
+

```

Comment:
minor knitpick: There seem to be way too many blank lines inserted in the source file. Can you remove those?